dhis-2/dhis-test-integration/src/test/java/org/hisp/dhis/trackedentity/TrackedEntityInstanceServiceTest.java

```diff
@@ -342,4 +345,45 @@ void testTrackedEntityInstanceGridWithNoDisplayAttributes()
         assertThrows( IllegalArgumentException.class,
             () -> entityInstanceService.getTrackedEntityInstancesGrid( params ) );
     }
+
+    @Test
+    void shouldCountOneEntityWhenOneAdded()
```

Comment:
`shouldCountOneEntityWhenOneIsPresent` makes more sense to me as right now the name suggests that you are testing a method that `adds` an entity.